public/css/main.css

```diff
@@ -20,14 +19,96 @@ a {
 }
 
 /*** GENERAL ***/
-
 body {
   background-color: #F2F2F2;
 }
 
 /*** HEADER BAR ***/
+.header {
+  background-color: #409A93;
+  font-family: 'Noto Sans', sans-serif;
+  padding: 2.5rem;
+  display: flex;
+  justify-content: space-between;
+}
+
+.naz-events{
+  color: white;
+  font-weight: bold;
+  font-size: 2.3rem;
+}
+
+.calendar {
+  font-size: 2.3rem;
+  color: white;
+  margin-top: 0.1rem;
+}
+
+.plus{
+  margin-top: 0.2rem;
+  font-size: 2.3rem;
+  color: white;
+}
+
+.languageNav {
+  background: #F2F2F2;
+  width: 100%;
+  padding: 0.6rem;
+  box-shadow: 0.4rem 0.5rem 0.5rem #DBDBDB;
+}
+
+.langBtn{
+  width: 48.5%;
+  font-weight: bolder;
+  font-size: 1.5rem;
+  color: #5C5C5C;
+  padding: 0.5rem;
+  background: #DBDBDB;
+  border: 0.1rem solid #DBDBDB;
+  border-radius: 0.1rem;
+  font-family: 'Lato', sans-serif;
+}
+
+#arLang{
+  margin-right: 0.2rem;
+  float: right;
+}
+
+#enLang {
+  margin-left: 0.2rem;
+}
 
 /*** EVENTS LIST ***/
+.container-div {
+  background-color: #EAEAEA;
+  padding: 0.6rem;
+  margin: 1rem;
+  box-shadow: 0.0rem 0.1rem 0.5rem #DADADA;
```

Comment:
I think this shadow could be a bit sharper - maybe:

`0 0.2rem 0.2rem #DADADA;`

Also note you don't need to specify a size unit when it's `0`